laevatein/src/main/java/com/laevatein/internal/utils/PhotoMetadataUtils.java

```diff
@@ -78,7 +82,19 @@ public static Point getBitmapSize(ContentResolver resolver, Uri uri, Activity ac
         return new Point((int) (w * widthScale), (int) (h * heightScale));
     }
 
-    public static Point getBitmapBound(ContentResolver resolver, Uri uri) {
+    public static Point safeGetBitmapBound(ContentResolver resolver, Uri uri) {
+        try {
+            return getBitmapBound(resolver, uri);
+        } catch (FileNotFoundException e) {
+            return null;
+        } catch (NullPointerException e) {
```

Comment:
This catch clause hides other faults, not only a failure in a specific device.
If we have some mistake in reading the image size then we should fix it first.